recipe/migrations/abseil_cpp202206230.yaml

```
@@ -0,0 +1,20 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 2
+
+# TODO: Make sure to remove the associated zip keys in the final file
+# when the migration is complete
+abseil_cpp:
+- '20220623.0'
+# new name, use both for now
+libabseil:
+- '20220623.0'
+# separate output for static builds (depending on C++ version)
+# also need to be migrated
+libabseil-static:
```

```suggestion
libabseil_static:
```

Fails with hyphen...